libretroshare/src/gxs/rsgxsrequesttypes.cc

```diff
@@ -23,6 +23,128 @@
 #include "rsgxsrequesttypes.h"
 #include "util/rsstd.h"
 
+std::ostream& operator<<(std::ostream& o,const GxsRequest& g)
```

Comment:
Maybe it is easier to just make them RsSerializable so they get << operator as a bonus?